src/test/groovy/org/gradle/android/WorkaroundTest.groovy

```diff
@@ -1,12 +1,17 @@
 package org.gradle.android
 
+import org.gradle.api.JavaVersion
+import org.junit.Assume
+import org.junit.experimental.categories.Category
 import spock.lang.Specification
 import spock.lang.Unroll
 
+@Category(MultiVersionTest)
```

Comment:
Is this a `MultiVersionTest` if it is only run against java8? Or does multiversion mean multiple android versions?

Reply:
There is a test in here for JDK8 and its associated versions and a test for JDK11 and its associated versions.